action/action-util/src/main/java/com/powsybl/action/util/OpenSwitchTask.java

```diff
@@ -7,17 +7,16 @@
 package com.powsybl.action.util;
 
 import com.powsybl.commons.PowsyblException;
-import com.powsybl.computation.ComputationManager;
-import com.powsybl.contingency.tasks.ModificationTask;
 import com.powsybl.iidm.network.Network;
 import com.powsybl.iidm.network.Switch;
+import com.powsybl.network.modification.NetworkModification;
 
 import java.util.Objects;
 
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
-public class OpenSwitchTask implements ModificationTask {
+public class OpenSwitchTask implements NetworkModification {
```

Comment:
Rename to OpenSwitch ?

Reply:
Yes, I was thinking of removing all the Task suffixes from the classes inheriting NetworkModification